website/vault.erb

```diff
@@ -422,7 +422,7 @@
                         </li>
 
                         <li<%= sidebar_current("docs-vault-resource-transform-template") %>>
-                            <a href="/docs/providers/vault/generated/resources/transform/template/name.html">vault_transform_template</a>
+                            <a href="/docs/providers/vault/r/transform_template.html">vault_transform_template</a>
```

Comment:
I wonder how the other links here are working since they're set to /vault/generated/resources/transform/...

Reply:
Good question, I guess that when the release is deployed to the registry there is some path munging going on. We should probably update all of the links at some point to be as the change above.

Of note, the links under `generated` are broken when testing the website with `make website-test`